rts/Rendering/GL/glExtra.cpp

```diff
@@ -517,8 +517,8 @@ void gleGenMeshBuffers(unsigned int* meshData) {
 	}
 
 	{
-		for (size_t n = 0; n < CYL_INDCS.size(); n++) { CYL_INDCS[n] += (BOX_VERTS.size()                   ); }
-		for (size_t n = 0; n < SPH_INDCS.size(); n++) { SPH_INDCS[n] += (BOX_VERTS.size() + CYL_VERTS.size()); }
+		for (auto& cyl_indc: CYL_INDCS) cyl_indc += BOX_VERTS.size();
```

Comment:
this might be an example where ranged loops express intent less clearly.

Reply:
I thought the same while I was writing it, so maybe `// NOLINT` it?

Then again it might just be something that gets clear once you get used to it. `For each element in $X: add to it`is almost like how'd you say it in English with a different word order.

Reply:
fwiw clang-tidy isn't producing any more warnings related to the checks I have enabled, so it's not like I would be spreading NOLINTs to more places (unless we decide to turn some other checks on). 
That said, yea, I'd prefer to use them to replace iterator based loops instead.